src/components/common/SearchBar/searchBar.tsx

```diff
@@ -181,6 +165,7 @@ const SearchBar = ({
     if (newValue.length) setErrorTooltip(false); //close the tooltip if there is at least 1 valid search term
     setValue(newValue);
     onChange_internal(newValue);
```

Comment:
Also, I think we can remove this line since the new line below covers its functionality

Reply:
@AbhiramTadepalli i think this is still relevant